1-index.html

```diff
@@ -44,13 +44,11 @@ <h2>Help needed?</h2>
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Jamulus is Open Source</h2>
-    <p>
-      Ideas? Found a bug? Want to contribute? Since Jamulus is open source you can help us! Find the project on <a href="https://sourceforge.net/projects/llcon/" target="_blank" rel="noreferrer">SourceForge</a> and the <a
-        href="https://github.com/corrados/jamulus" target="_blank" rel="noreferrer">source code on GitHub</a>!
-      Jamulus is licensed under the (<a href="http://www.gnu.org/licenses/gpl-2.0.html" target="_blank" rel="noreferrer">GPL,
-        GNU General Public License</a>).
-    </p>
+      <h2>Jamulus is <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">Free</a></h2>
```

Comment:
Could you please remove the link in the heading? It would be a duplicate. I'm not yet happy with the heading "Jamulus is free" since it might evoke wrong impressions. We could also rename it to something like "Jamulus is yours" or "Contribute". The term "Open Source" should be on the page since many people don't care about the difference and search for Open Source instead of "Free Software"

Reply:
OK, I'm pushing the changes…